### ISSUE TITLE
BUG: UUID not valid and appears as JSON in Cassandra (ScyllaDB)

### ISSUE THREAD

**Issue by yngfoxx** [beekeeper-studio/beekeeper-studio#2295]
**Describe the bug**
All UUIDs appear as a JSON with buffer as the only field in it.

**To Reproduce**
1. Spin up a ScyllaDB container
2. Connect to ScyllaDB using cqlsh
3. Create a keyspace and table with a UUID column as the pk
4. Add a row to the table, then delete it
5. Connect beekeeper to the ScyllaDB container
6. Attempt to delete a row from the keyspace & table created in Step 3

**Expected behavior**
It should view UUIDs as `e43322e9-aff9-4178-b4e3-881bbad20994` instead of this `{"buffer":"e43322e9aff94178b4e3881bbad20994"}`

**Screenshots / Gifs**
![image](https://github.com/beekeeper-studio/beekeeper-studio/assets/47150226/522db788-108f-4f88-a2c4-929a6c42998b)

**Version Information (please complete the following information):**
 - OS: Windows 11 Pro
 - App Version: Beekeeper Studio Team 4.6.4
 - Database type and version: ScyllaDB v5.4.6-0.20240418.10f137e367e3 (Cassandra Plugin)

**Comment by not-night-but** [comment]
Yeah this seems to be an issue with certain dbs. I've noticed it as well while developing for mongo. Should be solved easily enough with @azmy60's new transcoder system, right?

**Comment by azmy60** [comment]
Yeah, each database doesn't have transcoders by default. So we'll need to add the existing buffer transcoder `GenericBufferTranscoder` to Cassandra and mongo, or make a new one that fits the issue :)

**Comment by yngfoxx** [comment]
Hey guys, it seems this bug has been resolved so I will be closing this issue.
Thank you.
